tests/telemetry_test.py

```diff
@@ -163,16 +163,20 @@ def __call__(self, *args, **kwargs):
 
 
 class TransportClient:
-    def __init__(self, response=None, force_error=False, error=elasticsearch.TransportError):
-        self._response = response
+    def __init__(self, responses=None, force_error=False, error=elasticsearch.TransportError):
+        self._responses = responses
         self._force_error = force_error
         self._error = error
 
     def perform_request(self, *args, **kwargs):
         if self._force_error:
             raise self._error
         else:
-            return self._response
+            if len(self._responses) > 0:
```

Comment:
Nit: You could slim this down to `if self._responses:`

Reply:
Done